.pending/improvements/modules/_4875-refactored-sup

```diff
@@ -0,0 +1 @@
+#4875 refactored supply keeper integration tests to use SimApp
```

Comment:
when I continue to refactor tests for different modules, should I link to same issue with different pending logs, link to the prs or update this pending log with bullets for each module refactored?

Reply:
What about creating a multiple line pending entry? Then you add a new line on each PR. Check the supply module PR for reference